examples/flutter_view/pubspec.yaml

```diff
@@ -1,18 +1,22 @@
 name: flutter_view
 description: A new flutter project.
 
+environment:
+  # The pub client defaults to an <2.0.0 sdk constraint which we need to explicitly overwrite.
```

Comment:
This is really strange we have a constraint like that. Could it be that it is 'older' pub clients that default to <2.0.0 sdk constraint, but we should not be relying on them?

Reply:
This was added [here](https://dart-review.googlesource.com/c/sdk/+/30961/5/CHANGELOG.md#129):

"""
There is now a default SDK constraint of `<2.0.0` for any package with no
existing upper bound. This allows us to move more safely to 2.0.0. All new
packages published on pub will now require an upper bound SDK constraint so
future major releases of Dart don't destabilize the package ecosystem.
"""

Reply:
Ah, good to know. Thanks!